setup.py

```diff
@@ -50,7 +50,7 @@
 ]
 
 setup(name='dark-matter',
-      version='1.0.50',
+      version='1.0.51',
```

Comment:
Yes, thanks :-)  I remembered it when I was in bed, and then forgot it again.....  Done.